docs/users/gettingstarted/MovingToTheApp.md

```diff
@@ -77,34 +97,10 @@ By default, Stardrop uses a subfolder of "Mods" in the game directory. To preser
 7. Uninstall SMAPI from your game folder (Uninstall instructions [:fontawesome-brands-windows:](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Windows#Uninstall) [:fontawesome-brands-linux:](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Linux#Uninstall)).
 8. For each folder in the new "Stardrop Installed Mods" folder, create a zipped folder containing it. These can then be [imported into the app](./DownloadAMod.md#adding-a-mod-to-the-library-manually).
 
-## Mod Organizer 2
-[Mod Organizer 2](https://www.nexusmods.com/skyrimspecialedition/mods/6194) is an alternative community-created mod manager used primarily for Bethesda games but also offers support for Stardew Valley. To add your mods downloaded into MO2 to the app, follow the steps below:
-
-??? note "Note - Mod Updates"
-    Importing mods using this method means they do not have any Nexus Mods data associated with them and cannot be checked for updates.
-
-1. In Mod Organizer 2, verify you are managing Stardew Valley, then click the folder icon above the mod list and select "Open Downloads Folder".
-    ![The Mod Organizer 2 UI showing the button to open the download folder](../images/MO2DownloadFolder.webp)
-2. Take note of the download folder location - you'll need this shortly.
-3. Uninstall SMAPI from your game folder ([Uninstall instructions](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Windows#Uninstall)).
-4. Open the Nexus Mods app and [add Stardew Valley](../gettingstarted/AddGames.md).
-5. In the Library, select "Get Mods: From Drive".
-6. Navigate to the download folder location found in Step 2.
-7. Press ++ctrl+"A"++ to select all mod archives in the folder.
-8. Click "Open" and wait for the app to import all the mods (this can take several minutes).
-9. You can now start adding these mods to your loadout.
-
 ## Manually Installed Mods
 !!! warning "Manual Mods will be deleted when uninstalling the app"
     If you had manually installed mods when managing the game, they are added to the loadout as a special mod. Deleting the loadout of uninstalling the app will remove all of these files and revert your game folder back to game files only. 
```

Comment:
It says here

```
If you had manually installed mods when managing the game, they are added to the loadout as a special mod.
```

But we no longer have the 'Game Files' mod visible from the UI, and have not for a year, which is what this is referring to. This should go to talking about 'External Changes' directly.

-----

This also has a typo 'Deleting the loadout of uninstalling the app'

Reply:
I'm not referring to the "Game Files" part, but I've made this more clearly describe External Changes